### PR TITLE
Correcting Typos

### PR DESCRIPTION
**Pull request by gep13**
@ferventcoder sorry, this should have really been done on my own fork, but I was half way through all the edits before I realised, and didn't want to lose all the changes, so thought I would create temp branch in this repo :-(

There were a few more changes than I thought there would be, so thought you would want to check them through before pulling them in.

### CODE REVIEW

File: src/chocolatey/infrastructure/tolerance/FaultTolerance.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ private static bool log_is_in_debug_mode()
         }
 
         /// <summary>
-        /// Tries an action the specified number of tries, warning on each failure and erroring on the last.
+        /// Tries an action the specified number of tries, warning on each failure and raises error on the last attempt.
```
Comment (ferventcoder) [inline src/chocolatey/infrastructure/tolerance/FaultTolerance.cs]: Haha. I'm not sure if error can be used as a verb but I always have!

Comment (ferventcoder) [inline src/chocolatey/infrastructure/tolerance/FaultTolerance.cs]: Haha. I'm not sure if error can be used as a verb but I always have!

Comment (gep13) [inline src/chocolatey/infrastructure/tolerance/FaultTolerance.cs]: `Erroring` definitely didn't feel right to me :-)

Perhaps `failure and raises an error on the last attempt`

Comment (ferventcoder) [inline src/chocolatey/infrastructure/tolerance/FaultTolerance.cs]: Haha. I'm not sure if error can be used as a verb but I always have!

Comment (ferventcoder) [inline src/chocolatey/infrastructure/tolerance/FaultTolerance.cs]: Grrr, stupid phone

Comment (gep13) [inline src/chocolatey/infrastructure/tolerance/FaultTolerance.cs]: hee hee, I was wondering why you were repeating yourself :smile_cat:

**Comment by ferventcoder** [comment]
This should be good to go, I'll test it later

**Comment by gep13** [comment]
Cool, let me know if there are any other issues with this PR.

**Pull request merged**